static/conformance-view.js

```diff
@@ -0,0 +1,314 @@
+// Copyright (c) 2012-2017, Rubén Rincón
+//
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//
+//     * Redistributions of source code must retain the above copyright notice,
+//       this list of conditions and the following disclaimer.
+//     * Redistributions in binary form must reproduce the above copyright
+//       notice, this list of conditions and the following disclaimer in the
+//       documentation and/or other materials provided with the distribution.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+// POSSIBILITY OF SUCH DAMAGE.
+
+define(function (require) {
+    "use strict";
+
+    var options = require('options');
+    var _ = require('underscore');
+
+    require('selectize');
+
+    var Compiler = require('./compiler');
+
+    var compilers = options.compilers;
+    var compilersById = {};
+    _.forEach(compilers, function (compiler) {
+        compilersById[compiler.id] = compiler;
+        if (compiler.alias) compilersById[compiler.alias] = compiler;
+    });
+
+    function Conformance(hub, container, state) {
+        this.container = container;
+        this.eventHub = hub.createEventHub();
+        this.domRoot = container.getElement();
+        this.domRoot.html($('#conformance').html());
+        this.selectorList = this.domRoot.find('.compiler-list');
+        this.addCompilerButton = this.domRoot.find('.add-compiler');
+        this.compileButton = this.domRoot.find('.compile');
+        this.editorId = state.editorid;
+        this.source = state.source || "";
+        this.currentExpandedSource = "";
+        this.nextSelectorId = 0;
+
+        this.maxCompilations = 6;
```

Comment:
I tried to make it a configurable number, but didn't get too far with that. I'll try again now see if I get any better results.
And yes, it is mostly there to avoid melting CE